### DISCUSSION TITLE
Support for ADS12 in the `adc_ads1x1x` driver

### DISCUSSION THREAD

**Discussion by fgervais** [zephyrproject-rtos/zephyr#44920] [Ideas]
I see that the `adc_ads1x1x` driver has been added quite lately.

Right now it seems to support ADS10 and ADS11.

Any plan to also support ADS12? I'm looking at the `ADS1219`.

**Comment by towen** [comment]
The register descriptions for these chips ([ADS1119](https://www.ti.com/lit/ds/symlink/ads1119.pdf?ts=1650442283085) vs [ADS1219](https://www.ti.com/lit/ds/symlink/ads1219.pdf?ts=1650442264444)) look identical to me (only difference seems to be on page 24 of both datasheets, where a few more bits are added to the data register) so there is a good chance that the only reason the driver doesn't claim support for the ads12 chips, is because the developer doesn't have any to test 😉

If you have one of these chips, please test it and let us know if it works 😁

**Reply by fgervais** [reply]
I think the `ADS1119` is not supported either. I'm looking at the `config` register definitions and they don't seem to match the datasheet.

**Reply by XenuIsWatching** [reply]
That has a different reg model than the ads101x and ads111x...
Please take a look at the ads1119 pull request https://github.com/zephyrproject-rtos/zephyr/pull/37397
